ee/frontend/mobile-replay/schema/mobile/rr-mobile-schema.json

```
@@ -986,6 +990,54 @@
             "required": ["height", "id", "type", "width"],
             "type": "object"
         },
+        "wireframeScreenshot": {
+            "additionalProperties": false,
+            "description": "a screenshot behaves exactly like an image, but it is expected to be a screenshot of the screen at the time of the event, when sent as a mutation it must always attached to the root of the playback, when sent as an initial snapshot it must be sent as the first or only snapshot so that it attaches to the body of the playback",
```

> when sent as an initial snapshot it must be sent as the first or only snapshot

What is meant here? It implies to me that snapshots contain other snapshots. Does this more accurately describe what's going on?
```suggestion
            "description": "a screenshot behaves exactly like an image, but it is expected to be a screenshot of the screen at the time of the event. When sent as a mutation it must always attached to the root of the playback, when sent as an initial snapshot it must be sent as the first or only node so that it attaches to the body of the playback",
```